commentsoutput.txt

```diff
@@ -0,0 +1,22 @@
+ comment_id |                                                                                         body                                                                                         | article_id |    author     | votes |     created_at      
```

Comment:
Best to add this file to your .gitignore - this shouldn't be included in finalised code